<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Userame</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif Collection</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Re-Enter Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserameSignup">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ReEnterPass">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassSignup">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailSignup">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Signup">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordStrength">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>345</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AlrAccount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>525</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Already have an account?</string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameNotice">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="EmailNotice">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CapsLock">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>460</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
